index.js

```diff
@@ -122,6 +122,8 @@ class Client {
       routingRules,
       manageResources;
 
+	  this.options = this.serverless.service.custom.client;
```

Comment:
Please help us evaluate the issue and your fix by doing the following:

- Rebase off of master
- Format this file so this line is aligned well
- Test with serverless 3
- provide us with a repo that has the minimal code to reproduce this issue 

I realize this is a very old PR and may not be needed anymore so please help us validate that by doing the above steps